libdieharder/Makefile.am

```diff
@@ -19,8 +19,8 @@ man3_MANS = libdieharder.3
 # Library definitions
 lib_LTLIBRARIES = libdieharder.la
 libdieharder_la_LIBADD = -lgsl -lgslcblas -lm
-libdieharder_la_LDFLAGS = -version-number @DIEHARDER_LT_VERSION@
-libdieharder_la_CFLAGS = -std=c99 -Wall -pedantic
+libdieharder_la_LDFLAGS = -no-undefined -version-number @DIEHARDER_LT_VERSION@
+libdieharder_la_CFLAGS = -std=gnu99 -Wall -pedantic
```

Comment:
aaah, is that intentional? I thought the source was C99; I presume clang does gnu99 well enough, but changing the standard from something standard to something extended feels bad. What's the reasoning here? I think this is something you'd want to document.

Reply:
Oh, I changed` -std=c99` to `-std=gnu99`? I was not aware of that. Probably it came from an external patch that I added.
Yes, it came with the eeaad20b8e690389c94d7b1a18f9d64c0eead909 cygwin patch by fd00.
I can try without.